sdk/python/feast/cli.py

```diff
@@ -120,6 +120,18 @@ def feature_set():
     pass
 
 
+def _get_labels_dict(label_str: str):
+    labels_dict = {}
+    labels_kv = label_str.split(",")
+    if label_str == "":
+        return labels_dict
+    if len(labels_kv) % 2 == 1:
```

Comment:
This should still raise a value error

Reply:
Updated.

Reply:
Please print out what the user entered as part of exception.